src/components/VmDetails/cards/DetailsCard/index.js

```diff
@@ -911,7 +911,7 @@ class DetailsCard extends React.Component {
                         />
                       </FieldRow>
                       {/* VCPU Topology */}
-                      <Row className={style['field-row']}>
+                      <Row className={style['field-row-divide']}>
```

Comment:
I suggest a second class to make the margin adjustments without loosing the `field-row` styles.

Something like `className={`${style['field-row']} ${style['group-divider']}`}`.

Maybe it looks more complex, but I think it's a bit more flexible.

Thoughts?

Reply:
I think the original and also this solution - both are okay.